### DISCUSSION TITLE
[Help]: Still scroll the page when swiping on mobile phone

### DISCUSSION THREAD

**Discussion by Frrede** [davidjerleke/embla-carousel#1017] [Help] [answered]
### Summary
Is it possible, that the user on a mobile phone can still scroll the page to the left/right even though the carousel is swiping?
If not it would prevent a user to be able to scroll to the right to reach other elements.

### If applicable, which variants of Embla Carousel are relevant to this question?
* [x]  embla-carousel **(Core)**
* [ ]  embla-carousel-react
* [ ]  embla-carousel-vue
* [ ]  embla-carousel-svelte
* [ ]  embla-carousel-autoplay
* [ ]  embla-carousel-auto-scroll
* [ ]  embla-carousel-solid
* [ ]  embla-carousel-auto-height
* [ ]  embla-carousel-class-names
* [ ]  embla-carousel-fade
* [ ]  embla-carousel-docs **(Documentation)**
* [ ]  embla-carousel-docs **(Generator)**

### Additional information
![swipe-scroll](https://github.com/user-attachments/assets/ded5e671-81b2-40f0-82d9-fd7f966c673d)

**Comment by lovrozagar** [comment]
Hi Frrede,

Yes, it’s possible. But you will then need to set dragFree: true in the Embla settings:

const embla = EmblaCarousel(document.querySelector('.embla'), {
  dragFree: true,
});

Let me know if that helps!

**Reply by Frrede** [reply]
Hey :) Thank you for your answer but I think I didn't make it clear enough.
I want the native scroll from the browser still to work even though the carousel is moved by the swipe. So I guess there is a `preventDefault()` somewhere and I would like to disabled that, so the page would still scroll

**Reply by davidjerleke** [reply, marked as answer]
@Frrede this is not possible to achieve at the time of writing. You'll have to wait for the upcoming feature `watchDragMove` before you can control this. Roadmap here:

- #841 

This discussion is a duplicate of:

- #749
